installation-scripts/install-mysql.sh

```diff
@@ -16,6 +16,7 @@ dpkg -i mysql-apt-config_0.8.18-1_all.deb <<EOF
 ok
 EOF
 apt-key adv --keyserver keyserver.ubuntu.com --recv-keys 467B942D3A79BD29
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys B7B3B788A8D3785C
```

Comment:
It's seems to be needed for the docker build. I got the following error:
```
47.66 W: GPG error: http://repo.mysql.com/apt/debian buster InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B7B3B788A8D3785C
47.66 E: The repository 'http://repo.mysql.com/apt/debian buster InRelease' is not signed.
```
I try if only the new one is needed.

Reply:
It's seems to be needed for the docker build. I got the following error:
```
47.66 W: GPG error: http://repo.mysql.com/apt/debian buster InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY B7B3B788A8D3785C
47.66 E: The repository 'http://repo.mysql.com/apt/debian buster InRelease' is not signed.
```
I try if only the new one is needed.

Reply:
Only the new one is needed.

Reply:
@meiserloh verified that this is the correct key:
https://repo.mysql.com/apt/ubuntu/conf/distributions